lib/help_scout.rb

```diff
@@ -97,6 +97,17 @@ def get_mailboxes
     get("mailboxes")
   end
 
+  # Public: Get ratings
+  #
+  # More info: http://developer.helpscout.net/help-desk-api/reports/user/ratings/
+  # 'rating' parameter required: 0 (for all ratings), 1 (Great), 2 (Okay), 3 (Not Good)
+  def get_ratings(mailbox_id, page = 1, modified_since = nil, user_id = nil, start_date = nil, end_date = nil)
+    rating = 0
+    query = {mailboxes: mailbox_id, user: user_id, start: start_date, end: end_date, rating: rating}
+
+    get("reports/user/ratings", page, modified_since, query)
```

Comment:
This is still your old Scoutest syntax, you need to move them to an options hash. 

I'd write the whole method like this.

Thoughts: name it like the end point, has all the required parameters as parameters in the method. Might want to cast the dates to the right format (ISO 8601, in UTC) if you want.

```ruby
def reports_user_ratings(user_id, rating, start_date, end_date, options)
  options = {
    user: user_id,
    rating: rating,
    start: start_date,
    end: end_date,
  }
  
  get("reports/user/ratings", options)
```

You can use all the extra parameters (`modified_since`, `user_id`, etc) via the options Hash in Scoutest.